service/controller/key/key.go

```diff
@@ -1 +1,88 @@
 package key
+
+import (
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"github.com/giantswarm/microerror"
+)
+
+const (
+	ViewAllRole = "view-all"
+)
+
+func NewViewAllRole(resources []*metav1.APIResourceList) (*rbacv1.Role, error) {
```

Comment:
This logic is not really "key" function stuff. Let's put it to the resource where it is used.